easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-3.1.0-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'MakeCp'
+
+name = 'HiC-Pro'
+version = '3.1.0'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'http://nservant.github.io/HiC-Pro'
+description = """HiC-Pro was designed to process Hi-C data, from raw fastq files (paired-end Illumina data) 
+ to the normalized contact maps."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://github.com/nservant/HiC-Pro/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['25a88d89813e8e9653f59ca0bd6eaad8b0c8dbb5e578e233f2b3db7b04256f2f']
+
+dependencies = [
+    ('R', '4.2.1'),
+    ('Bowtie2', '2.4.5'),
+    ('Pysam', '0.19.1'),
+    ('SAMtools', '1.16.1'),
+    ('bx-python', '0.9.0'),
+    ('iced', '0.5.10'),
+]
+
+skipsteps = ['configure']
+
+prebuildopts = 'sed -i "s|PREFIX = |PREFIX = %(installdir)s|" config-install.txt && '
```

Comment:
Could you add a quick summary of what's going on here, for the benefit of future maintainers?

Reply:
As a comment, I mean.